test/units/testsuite-57-short-lived.sh

```diff
@@ -12,7 +12,7 @@ fi
 echo "$counter" > /tmp/testsuite-57.counter
 
 if [ "$counter" -eq 5 ] ; then
-    systemctl kill --kill-who=main -sUSR1 testsuite-57.service
+    systemctl kill --kill-whom=main -sUSR1 testsuite-57.service
```

Comment:
maybe leave one behind, so that we really ensure we don't break backward compat?

Reply:
Done. I also fixed the getopt/popt mixup.